lib/cn/omf.h

```diff
@@ -175,12 +175,12 @@ static_assert(HBLOCK_HDR_PAGES == 1, "Hblock header spanning more than 1 page ha
 #define VGROUP_MAP_MAGIC   ((uint32_t)('v' << 24 | 'g' << 16 | 'p' << 8 | 'm'))
 
 struct vgroup_map_entry_omf {
-    uint32_t vgme_vbidx;
-    uint32_t vgme_vbadj;
+    uint16_t vgme_vbidx;
+    uint16_t vgme_vbadj;
 } HSE_PACKED;
 
-OMF_SETGET(struct vgroup_map_entry_omf, vgme_vbidx, 32)
-OMF_SETGET(struct vgroup_map_entry_omf, vgme_vbadj, 32)
+OMF_SETGET(struct vgroup_map_entry_omf, vgme_vbidx, 16)
+OMF_SETGET(struct vgroup_map_entry_omf, vgme_vbadj, 16)
```

Comment:
Hmm, an odd number of these will leave us on a 4-byte boundary and leave all 64-bit fields that follow unaligned...
I am thinking maybe we should just make all our omf structures packed(8).

Reply:
Seems reasonable to me

Reply:
Agreed. However, this case should be fine as the vgmap is on a page of its own in the hblock and the section that follows it starts on a page boundary.